_config.yml

```diff
@@ -18,25 +20,25 @@ collections:
     permalink: "/blog/:categories/:year/:month/:day/:title.html"
     
 defaults:
-  - scope:
-      path: "" # an empty string here means all files in the project
-    values:
-      layout: "default"
+  # - scope:
+  #     path: "" # an empty string here means all files in the project
+  #   values:
+  #     layout: "default"
```

Comment:
Yup, I was trying to stop Jekyll from processing the gulp stuff, probably could be deleted